modules/packages/LinearAlgebra.chpl

```diff
@@ -2063,6 +2321,157 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
   return C;
 }
 
+/*
+  Matrix exponential using Pade approximation. This method returns N*N matrix which
+  is Matrix exponential of `A`
+
+  Args:
+  A: Expects an N*N square matrix
+  UseExactOneNorm: boolean value specifying if the onenorm has to be exact.
```

Comment:
This one will need to be updated.  It should be e.g. `:arg A:`, as demonstrated on [the argument section of the chpldoc page](https://chapel-lang.org/docs/latest/tools/chpldoc/chpldoc.html#argument-and-return-yield-types)